python/paddle/trainer_config_helpers/tests/configs/check.md5

```diff
@@ -0,0 +1,3 @@
+9c038249ec8ff719753a746cdb04c026  layer_activations.protostr
```

Comment:
@emailweixu I just generate proto string and check the md5sum same.

I read the proto string manually to check proto string correctly. Is this way ok?